### PR TITLE
fix(recommend): remove unused trendingFacets props 

### PR DESCRIPTION
**Pull request by marialungu**
Remove `fallbackParameters` and `queryParameters` given that the recommend API does not support these props.
🐛 : [JIRA](https://algolia.atlassian.net/browse/RECO-1173)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,5 @@ dist
 *.umd.js
 yarn-error.log
 .env
+
+.idea/
```
Comment (jojva) [inline .gitignore:10]: Make a separate PR for this

Comment (Haroenv) [reply .gitignore:10]: it probably should just be reverted actually, local files should be in a global gitignore

**Review by Haroenv — COMMENTED** [review]
change itself looks legit

File: packages/recommend/src/types/TrendingFacetsQuery.ts [hunk 2]
```diff
@@ -1,3 +1,6 @@
 import { TrendingQuery } from './TrendingQuery';
 
-export type TrendingFacetsQuery = Omit<TrendingQuery, 'model' | 'facetValue'>;
+export type TrendingFacetsQuery = Omit<
+  TrendingQuery,
+  'model' | 'facetValue' | 'fallbackParameters' | 'queryParameters'
```
Comment (francoischalifour) [inline packages/recommend/src/types/TrendingFacetsQuery.ts:5]: At this point why have a shared `TrendingQuery` type if they differ so much?

According to the API endpoints:

- Trending facets
  - `facetName`: required
- Trending items
  - `facetName`: optional
  - `facetValue`: optional

Comment (jojva) [reply packages/recommend/src/types/TrendingFacetsQuery.ts:5]: Yeah that's a good point

Comment (marialungu) [reply packages/recommend/src/types/TrendingFacetsQuery.ts:5]: yeah, i wanted to do that in a separate PR, both for client and for the libs

**Review by jojva — APPROVED** [review]
